src/core/constants.py

```diff
@@ -24,10 +23,30 @@
 DB_RELAY_DIR = Path(HOME_DIR, '.local/share/polkadot/polkadot')
 WASM_DIR = Path(HOME_DIR, 'wasm')
 
-SNAP_SERVICE_NAME = "snap.polkadot.polkadot"
-SNAP_BINARY = "polkadot.polkadot-cli"
-SNAP_NODE_KEY_FILE = Path(SNAP_COMMON_DIR, 'node-key')
-SNAP_CHAIN_SPEC_DIR = Path(SNAP_COMMON_DIR, 'spec')
-SNAP_DB_CHAIN_DIR = Path(SNAP_COMMON_DIR, 'polkadot_base', 'chains')
-SNAP_DB_RELAY_DIR = Path(SNAP_COMMON_DIR, 'polkadot_base', 'polkadot')
-SNAP_WASM_DIR = Path(SNAP_COMMON_DIR, 'wasm')
+SNAP_USER = 'root'
+SNAP_CONFIG = {
+    'polkadot': {
+        'base_path': '/var/snap/polkadot/common/polkadot_base',
+        'snap_name': 'polkadot',
+        'service_name': 'polkadot',
+        'cli_command': 'polkadot.polkadot-cli',
+        'snap_binary_path': '/snap/polkadot/current/bin/polkadot',
+        'chain_spec_dir': Path('/var/snap/polkadot/common/polkadot_base/spec'),
+        'chain_db_dir': Path('/var/snap/polkadot/common/polkadot_base/chains'),
+        'relay_db_dir': Path('/var/snap/polkadot/common/polkadot_base/polkadot'),
+        'wasm_dir': Path('/var/snap/polkadot/common/polkadot_base/wasm'),
+        'node_key_file': Path('/var/snap/polkadot/common/node-key')
+    },
+    'polkadot-parachain': {
+        'base_path': '/var/snap/polkadot-parachain/common/polkadot_base',
+        'snap_name': 'polkadot-parachain',
+        'service_name': 'polkadot-parachain',
+        'cli_command': 'polkadot-parachain.cli',
+        'snap_binary_path': '/snap/polkadot-parachain/current/bin/polkadot-parachain',
```

Comment:
Perhaps make all paths `Path`?

```suggestion
        'base_path': Path('/var/snap/polkadot-parachain/common/polkadot_base'),
        'snap_name': 'polkadot-parachain',
        'service_name': 'polkadot-parachain',
        'cli_command': 'polkadot-parachain.cli',
        'snap_binary_path': Path('/snap/polkadot-parachain/current/bin/polkadot-parachain'),
```